src/main/kotlin/MainController.kt

```diff
@@ -48,4 +48,8 @@ class MainController : Controller() {
         selectedWindow.value.connection?.joinChannel(value) ?: return
     }
 
+    fun leaveChannel(name: String) {
+        selectedWindow.value.connection?.leaveChannel(name) ?: return
```

Comment:
No need for the return `?: return`, AFAIK? (Same above as well)

```suggestion
        selectedWindow.value.connection?.leaveChannel(name)
```